karma.conf.js

```diff
@@ -16,7 +16,7 @@ module.exports = (config) => {
       clearContext: false // leave Jasmine Spec Runner output visible in browser
     },
     coverageIstanbulReporter: {
-      dir: require('path').join(__dirname, './coverage/vital-app'),
+      dir: require('path').join(__dirname, './coverage/vital-app/src/app'),
```

Comment:
Any specific need for this change? I'm curious.

Reply:
We are interested in getting coverage for just our app. I don't want it scan outside app folder.